Models/Index4Response.js

```diff
@@ -157,7 +158,7 @@ static _mackWebpAsync(content, quality) {
   static _createDirectoryIfNotExist(filePath) {
     const pathDirectory = path.dirname(filePath);
     if (!fs.existsSync(pathDirectory)) {
-      fs.mkdirSync(pathDirectory);
+      fs.mkdirSync(pathDirectory,{recursive : true});
     }
```

Comment:
Why should we not use `Index4Response._createDirectoryIfNotExist(pathDirectory)`?